code/parse/parselo.cpp

```diff
@@ -1245,8 +1257,8 @@ void stuff_string(char *outstr, int type, int len, char *terminators)
 	if (type == F_FILESPEC) {
 		// Make sure that the passed string looks like a good filename
 		if (strlen(read_str) == 0) {
-			// Empty file name is not valid!
-			error_display(1, "A file name was expected but no name was supplied!\n");
+			// Empty file name is probably not valid!
+			parse_log("A file name was expected but no name was supplied! This is probably a mistake.");
```

Comment:
So my only question is why this shouldn't also be an `error_display(0, ...)` call, since there shouldn't be any reason for anyone actively developing a mod to leave a filename field present but empty (that I can think of).